rpc_util.go

```diff
@@ -217,6 +217,18 @@ func Code(err error) codes.Code {
 	return codes.Unknown
 }
 
+// Desc returns the error description of err if it was produced by the rpc system.
+// Otherwise, it returns err.Error() or empty string when err is nil.
+func Desc(err error) string {
```

Comment:
This isn't a great name in the context. `grpc.Desc` doesn't say much (whereas at least the above `grpc.Code` is referring to the more specific "code" concept).

Perhaps `ErrorDesc` would be clearer?